resources/ts/Components/IntegrationCard.tsx

```diff
@@ -38,17 +39,63 @@ export const OpenWidgetBuilderButton = ({
   }
 
   return (
-    <ButtonLinkSecondary href={`/integrations/${id}/widget`} target={"_blank"}>
+    <ButtonLinkSecondary
+      href={`/integrations/${id}/widget`}
+      target={"_blank"}
+      className="flex self-start"
+    >
       {t("integrations.open_widget")}
     </ButtonLinkSecondary>
   );
 };
 
-export const IntegrationCard = ({ id, name, type, status, onEdit }: Props) => {
+export const IntegrationCard = ({
+  id,
+  name,
+  type,
+  status,
+  credentials,
+  onEdit,
+}: Props) => {
   const { t } = useTranslation();
 
   const integrationTypesInfo = useIntegrationTypesInfo();
 
+  const auth0TestClient = credentials.auth0.find(
+    (client) => client.integrationId === id && client.tenant === "test"
```

Comment:
Maybe it's interesting to make a const with the different environments?
In case you need it in other places?  Or are the tenants already correctly typed?

Something like:

```typescript
const Environments = {
 Test: 'test',
 Acceptance: 'acc',
 Production: 'prod'
} as const;
```

Reply:
Yes that constant already exists. and the type of tenant is also correct based on the constant.
So both are correct, it's a matter of preference using literal strings or the constant